### ISSUE TITLE
Have both the CLI and GUI version of OpenCOR to build at the same time on Windows

### ISSUE THREAD

**Issue by agarny** [opencor/opencor#343]
Right now, the CLI and GUI versions of OpenCOR are built individually. This can be a bit annoying when wanting to run the tests using Qt Creator on Windows since we will first have to build the CLI version before we can run the tests. So, nothing critical, just that it would be nice to have both versions built for us together.
